src/driver.coffee

```diff
@@ -51,11 +48,11 @@ class Neo4jDB
     else
       "#{value}"
 
-  reset: ->
-    console.log "Resetting Neo4j"
+  reset: () ->
+    console.log "Resetting Neo4j..."
+    @query "MATCH (n) OPTIONAL MATCH (n)-[r]-() DELETE n,r"
```

Comment:
I guess this wont fail is theres nothing in it?

I think this should do it thought: `"MATCH ()-[r]->(), (n) DELETE r,n"`

Also, I think it would be better not to use these. I've crashed Neo4j when trying to delete 100s or thousands of items this way. Its much better to delete at the commandline

the right way to do this is to integrate with the meteor CLI (currently no public API). That way, you coudl create a DB in `.meteor/local` for each project.